Makefile

```
@@ -240,7 +240,7 @@ docker-push:
 
 # Build and push the dev image
 .PHONY: docker-dev-push
-docker-dev-push: build image-hive-dev docker-push
+docker-dev-push: build image-hive docker-push
```

```suggestion
docker-dev-push: build image-hive-fedora-dev docker-push
```

My mistake, thanks for catching. It's actually supposed to be the above.

My workflow now is:

```
$ time make build image-hive-fedora-dev docker-push && kubectl delete pod -n hive --all
```

---

What's the difference between `image-hive` and `image-hive-fedora-dev`? I am a macOS user, so I want to make sure this works on both Linux and macOS.

---

image-hive is a slow, but full build of a fresh container. fedora-dev is using the locally build binaries and go cache, and injecting those binaries into a pre-made base layer for faster development. 

on macos, i think you want to avoid all the fedora/dev targets and just use make image-hive. (build not required)

---

Thanks, @dgoodwin. Updated the line and rebased. PTAL